src/System Application/App/AI/src/Copilot/CopilotCapabilityImpl.Codeunit.al

```
@@ -277,6 +277,10 @@ codeunit 7774 "Copilot Capability Impl"
     begin
         CopilotCapability := Enum::"Copilot Capability".FromInteger(Capability);
 
+        if (CopilotCapability <> Enum::"Copilot Capability"::Chat) or
```

Not sure if we should throw an error here.

---

Feel free to  resolve it if there is reason we cannot throw error from here

---

We should not be throwing an error. The subscriber should not handle this if it is not of these types